packages/cli-helpers/package.json

```diff
@@ -35,7 +35,8 @@
     "pascalcase": "1.0.0",
     "prettier": "2.8.8",
     "prompts": "2.4.2",
-    "terminal-link": "2.1.1"
+    "terminal-link": "2.1.1",
+    "toml": "3.0.0"
```

Comment:
Is there any need for this `toml` package specifically? In all other packages we use `@iarna/toml` and it might be nice just to keep it consistent?